webextension/scripts/popup.js

```diff
@@ -208,6 +208,20 @@ function loginSuccess() {
   $('#login-tab-btn').hide()
   $('#logout-tab-btn').css('display', 'inline-block')
 
+  getUserInfo().then(info => {
+    if (info && ('screenname' in info) && ('itemname' in info)) {
+      var previous_user_itemname = ''
+      chrome.storage.local.get(['itemname'], (settings) => {
+        if (settings && settings.itemname) {
+          previous_user_itemname = settings.itemname
+        }
+      }) 
+      if (previous_user_itemname != info.itemname){
```

Comment:
This if statement is not guaranteed to run after the previous assignment to `previous_user_itemname` since that assignment is in an async block.